### ISSUE TITLE
Cannot set kf.indicatorType in swift 3

### ISSUE THREAD

**Issue by AndrewBarba** [onevcat/Kingfisher#469]
See below:

<img width="956" alt="screen shot 2016-10-05 at 12 43 14 pm" src="https://cloud.githubusercontent.com/assets/1316152/19122657/7a8e6fca-8af9-11e6-8f1f-122a22f46a2e.png">

It makes sense, since `kf` is a struct, setting a property would have to change `kf` on the image view but I believe you only have a getter for that.

Using `kf_indicatorType = .activity` continues to work but shows the deprecation warning

**Comment by AndrewBarba** [comment]
I should have been more clear, this does not compile:

```
let imageView = UIImageView()
imageView.kf.indicatorType = .activity
```

But this does:

```
var imageView = UIImageView()
imageView.kf.indicatorType = .activity
```

The screenshot I posted is my own extension on `UIImageView` which I guess is the same thing as the `let` case

**Comment by onevcat** [comment]
:( You are right. I will take a look to see what we could do for it.

**Comment by onevcat** [comment]
Fixed in #470
